testing/single/src/docstrings.py

```diff
@@ -0,0 +1,1295 @@
+# This file is for testing different types of docstrings and verifying they display
+# correctly.
+# Try it with `python.analysis.supportRestructuredText` both true and false. 
+import numpy as np
```

Comment:
Should `numpy` also be added to requirements.txt?

Reply:
Ah good idea. I'll do that.